tests/test_parser.py

```diff
@@ -1961,7 +1961,7 @@ def test_import_custom_sources(self):
 
             p = _Lark(grammar, import_paths=[custom_loader])
             self.assertEqual(p.parse('ab'),
-                             Tree('start', [Tree('startab', [Tree('ab__expr', [Token('ab__A', 'a'), Token('ab__B', 'b')])])]))
+                             Tree('start', [Tree('startab', [Tree('ab__expr', [Token('AB__A', 'a'), Token('AB__B', 'b')])])]))
```

Comment:
Btw, this is also a change I made. Should this stay changed, or should this be reverted? My goal here was to make the mangled names valid terminal names.

Reply:
I think it's better to preserve the case of the module name. It will be easier for debugging.

The letter-case of symbols is only used as part of the grammar language, and isn't a necessity for any other part of Lark.

Also, we might change that in the future, so that terminals are *any* symbol with an upper-case letter, to allow `TerminalsLikeThis`.

Reply:
The problem with this is that it is then `term.isupper()` will no longer hold true. This can however be fixed by just looking at the part after the last `__`.